pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTenantRestletResource.java

```diff
@@ -690,9 +686,46 @@ public SuccessResponse deleteTenant(
   @ApiOperation(value = "Rebalances all the tables that are part of the tenant")
   public TenantRebalanceResult rebalance(
       @ApiParam(value = "Name of the tenant whose table are to be rebalanced", required = true)
-      @PathParam("tenantName") String tenantName, @ApiParam(required = true) TenantRebalanceConfig config) {
+      @PathParam("tenantName") String tenantName,
+      @ApiParam(value = "Number of table rebalance jobs allowed to run at the same time", required = true, example =
+          "1")
+      @QueryParam("degreeOfParallelism") Integer degreeOfParallelism,
+      @ApiParam(value =
+          "Comma separated list of tables (with OFFLINE or REALTIME suffix) that are allowed in this tenant rebalance"
```

Comment:
nit: Re-word to: "Comma separated list of tables with type that are allowed in this tenant rebalance"
Maybe you can provide an example?

Please update the other parameters as well, i.e. `blockTables`